index.js

```diff
@@ -286,6 +287,9 @@ demo`;
         invalid(`Invalid action "${action}" for modules command`);
     }
   },
+  publish: () => {
+    publish();
+  },
```

Comment:
I know the ticket was like this and we will leave it as is, but just for record purposes:
We probably want this action to be` modules publish`. Kind of like grouping module related actions that make sense.
We can revisit these commands later, however